File: Assignment 2/if else/Greateroftwo.cpp
#include<iostream>
using namespace std;
class Greatest
{
    int number1,number2;
    public: void great()
    {
        cout<<"Enter the two numbers:\n";
        cin>>number1>>number2;
        if(number1>number2)
        {
            cout<<"First number is the greatest";
        }
        else
        {
            cout<<"Second number is the greatest";
        }
    }
};
int main()
{
    Greatest obj;
    obj.great();
}

File: Java assignment(1-2)/java Assignment 2/GreaterOfTwo.java
import java.util.Scanner;
class GreaterOfTwo
{
int num1,num2;
void getData()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter two numbers:");
num1=sc.nextInt();
num2=sc.nextInt();
if(num1>num2)
{
System.out.println("First number is greater");
}
else
{
System.out.println("Second number is greater");
}
}
public static void main(String...args)
{
GreaterOfTwo obj=new GreaterOfTwo();
obj.getData();
}
}